src/RepositorySelect/usePresetBuildOptionStore.ts

```diff
@@ -54,7 +54,7 @@ export const usePresetBuildOptionStore = create<PresetBuildOptionStore>()(
         try {
           const project = await api.getProject(repoName);
           const newProjectName = !!project
-            ? `${repoName}-${nanoid(7)}`
+            ? `${repoName}-${nanoid(7)}`.toLowerCase()
```

Comment:
nanoid를 lowercase로 변경하면, 현실적으로는 크게 문제없이 동작할 것 같기는한데 이게 고르게 랜덤으로 분포되는 데이터는 아닐 수도 있을 것 같아요!

nanoid를 custom으로 채울 수 있는 메소드도 있는데 이걸 활용해봐도 괜찮을 것 같습니다~
```suggestion
          const randomName = customAlphabet("0123456789abcdefghijklmnopqrstuvwxyz", 7);
            ? `${repoName}-${randomName}`
```

Reply:
nanoid 뿐만 아니라 reponame도 소문자로 변경이 되어야하는 부분이었는데 적용이 다 안되어있었네요~ 추가로 수정해놓을게요~